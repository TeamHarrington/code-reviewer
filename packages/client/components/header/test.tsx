import { Header } from '.'
import renderer from 'react-test-renderer'
import React from 'react'

test('Header Component', () => {
  const component = renderer.create(<Header />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
