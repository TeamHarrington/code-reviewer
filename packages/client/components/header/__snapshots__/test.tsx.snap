// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component 1`] = `
<div
  className="header__Container-sc-1vegvik-0 gHuXxp"
>
  <a
    href="app url, TBD"
  >
    <div
      className="header__LogoContainer-sc-1vegvik-3 fDrKtj"
    >
      <svg
        height={24}
        viewBox="0 0 60 60"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          stroke="none"
          strokeWidth="1"
          transform="translate(-466 -1076) translate(466 1076)"
        >
          <path
            d="M1 0h29c16.569 0 30 13.431 30 30 0 16.569-13.431 30-30 30H1a1 1 0 01-1-1V1a1 1 0 011-1z"
            fill="#004DCA"
          />
          <rect
            fill="#181196"
            height="60"
            rx="1"
            width="22.388"
            x="0"
            y="0"
          />
          <path
            d="M2 6.94a18.26 18.26 0 0111.417-1.656c9.855 1.741 16.464 10.99 14.762 20.66-.764 4.336-3.085 8.037-6.316 10.632l.64.863 2.17.383L36 53.71 31.203 57l-11.36-15.865.375-2.128-.609-.885A18.236 18.236 0 017.253 40.3 18.438 18.438 0 012 38.532v-6.604a11.96 11.96 0 008.021 3.064c6.64 0 12-5.36 12-12s-5.36-12-12-12a11.96 11.96 0 00-8.02 3.065z"
            fill="#FFF"
            fillRule="nonzero"
          />
          <circle
            cx="10"
            cy="23"
            fill="#FFF"
            fillRule="nonzero"
            r="9"
          />
        </g>
      </svg>
    </div>
  </a>
  <div
    className="header__Title-sc-1vegvik-4 fFqmux"
  >
    
  </div>
  <div
    aria-haspopup="menu"
    aria-label="Hello . Click to open user menu."
    className="header__UserContainer-sc-1vegvik-5 hwJtvu"
    onClick={[Function]}
    role="button"
    tabIndex={0}
  >
    <div
      className="header__UserNameContainer-sc-1vegvik-1 fHncqZ"
    >
      
    </div>
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
      />
    </svg>
  </div>
</div>
`;
